src/app/examples/form/components/form.component.html

```diff
@@ -0,0 +1,119 @@
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <h1 class="main-heading">{{ 'anms.examples.form.title' | translate }}</h1>
+    </div>
+  </div>
+  <form [formGroup]="form">
+    <div class="row">
+      <div class="col-md-6" [ngClass]="routeAnimationsElements">
+        <h2>Contact form <mat-slide-toggle formControlName="autosave">Auto Save</mat-slide-toggle></h2>
```

Comment:
Let's implement all strings using the i18n (so similarly to `'anms.examples.form.title'`), it is ok to only provide english translations, it will default to english for other languages and they can be added later.